### ISSUE TITLE
[bug] /admin/syncthing brings you to the dashboard page

### ISSUE THREAD

**Issue by fenpaws** [mickael-kerjean/filestash#284]
### description of the bug: 
forward of admin/synchting not working, goes back do /admin/dashboard

### step by step instructions to reproduce the bug:
1. add syncthing url (in my case http://syncthing:8384)
2. access it via /admin/syncthing

### observed behavior:
forward to admin/synchting brings you back to /admin/dashboard

### expected behavior:
displays the syncthing UI

### addidional information
I used the LinuxServers docker version of syncthing. I can access it normally without any problems, I included it within the provided [docker-compose](https://gitlab.com/fenpaws/docker-compose/-/blob/devel/simple-sync/docker-compose.yml) file of filestash.

**Comment by mickael-kerjean** [comment]
I was able to replicate the issue you're seeing only when changing the setting and not restart the instance. Some options such as this one need a restart to be effective

**Comment by fenpaws** [comment]
Thanks for the reply, it works after a full restart of the container. But now I get a Login prompt when I try to go to Syncthing. The only issue is that I don't have one set for Syncthing, but nothing i cant fix.

Even then it always prompts me to log in, but the issue now is that I won't let me. It just reloads again.

The same creds work directly and without issues in Syncthing.

**Comment by mickael-kerjean** [comment]
> But now I get a Login prompt when I try to go to Syncthing

from `/admin/syncthing` you should be able to login with the username 'admin' and password to be your admin password of the Filestash instance.

> The only issue is that I don't have one set for Syncthing, but nothing i cant fix.

To me knowledge syncthing doesn't have users, that's why this feature was created in the first place, aka safely expose your syncthing instance over the internet without having to configure a reverse proxy and manage yet another SSL certificate.

There's plan to get deeper integration but I lack the time to work on it.

**Comment by fenpaws** [comment]
I tired the creds for the Filestash instance but I get the same issue there.

And syncthing has a basic user auth system in place now
<img width="893" alt="grafik" src="https://user-images.githubusercontent.com/29946173/84957387-60785480-b0fb-11ea-9a6d-52d23c0f07b9.png">

awesome that you want to create a deeper integration for it, I'm all for this idea! I use your tool for easy cloud access to my files. So I don't need to use Owncloud or Nextcloud. Keep up the great work!

**Comment by mickael-kerjean** [comment]
> syncthing has a basic user auth system in place now

That wasn't the case when I made that integration, that's good to know they added the feature.

**Comment by fenpaws** [comment]
Okay after leaving this issue for a bit I decided to try it again, and I have a solution for the login problem. It's really simple, just use the same creds for filestash as for syncthing.

After I edited my syncthing login to reflect the file stash login it works like a charm.
User for Syncthing and filestash:`admin` 
Password for Syncthing and filestash:`***` 


I would suggest getting rid of the login prompt inside of filestash since syncthing handles this on its own, or to add an option to enable the login if the user decides to.
